internal/service/tesla_fleet_api_service.go

```diff
@@ -80,21 +80,23 @@ type fleetStatusResponse struct {
 	KeyPairedVINs []string `json:"key_paired_vins"`
 	UnpairedVINs  []string `json:"unpaired_vins"`
 	VehicleInfo   map[string]struct {
-		FirmwareVersion                string `json:"firmware_version"`
-		VehicleCommandProtocolRequired bool   `json:"vehicle_command_protocol_required"`
-		DiscountedDeviceData           bool   `json:"discounted_device_data"`
-		FleetTelemetryVersion          string `json:"fleet_telemetry_version"`
-		TotalNumberOfKeys              int    `json:"total_number_of_keys"`
+		FirmwareVersion                    string `json:"firmware_version"`
+		VehicleCommandProtocolRequired     bool   `json:"vehicle_command_protocol_required"`
+		DiscountedDeviceData               bool   `json:"discounted_device_data"`
+		FleetTelemetryVersion              string `json:"fleet_telemetry_version"`
+		TotalNumberOfKeys                  int    `json:"total_number_of_keys"`
+		SafetyScreenStreamingToggleEnabled *bool  `json:"safety_screen_streaming_toggle_enabled"`
 	} `json:"vehicle_info"`
 }
 
 type VehicleFleetStatus struct {
-	KeyPaired                      bool
-	VehicleCommandProtocolRequired bool
-	FirmwareVersion                string
-	DiscountedDeviceData           bool
-	FleetTelemetryVersion          string
-	NumberOfKeys                   int
+	KeyPaired                          bool   `json:"key_paired"`
+	VehicleCommandProtocolRequired     bool   `json:"vehicle_command_protocol_required"`
+	FirmwareVersion                    string `json:"firmware_version,omitempty"`
+	DiscountedDeviceData               bool   `json:"discounted_device_data"`
+	FleetTelemetryVersion              string `json:"fleet_telemetry_version,omitempty"`
+	NumberOfKeys                       int    `json:"number_of_keys,omitempty"` // limit 20
+	SafetyScreenStreamingToggleEnabled *bool  `json:"safety_screen_streaming_toggle_enabled,omitempty"`
```

Comment:
suggestion: For most of our APIs we use camelCase so it's easier on people like mobile if they know what to expect, That being said tesla-oracle is not part of the protocol so it can differ